src/components/Events.tsx

```diff
@@ -0,0 +1,79 @@
+import Link from 'next/link'
+import React from 'react'
+import type {EventData} from '../types/index'
+import DisplayRC from './DisplayRC'
+
+const events: EventData[] = [
+    {
+        "id": 1,
+        "title": "Physical Meetups",
+        "description": "Physical meetups where we share, learn and network.",
+        "image_url": "https://bit.ly/placeholder-img",
+        "target": "https://kommunity.com/reactjs-developer-community-kenya-reactdevske/events"
+      },
+      {
+        "id": 2,
+        "title": "Weekly Online Standups",
+        "description": "Weekly open calls in the community’s Telegram group discussing industry-related topics.",
+        "image_url": "https://bit.ly/placeholder-img",
+        "target": "https://t.me/joinchat/eNHADkJLCaMzZTM8"
```

Comment:
This is not the correct link, the link in the `events.json` is `https://bit.ly/joinreactdevske`.